nms/app/packages/magmalte/app/views/equipment/FEGGatewayDetailStatus.js

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright 2020 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * @flow strict-local
+ * @format
+ */
+import type {DataRows} from '../../components/DataGrid';
+
+import DataGrid from '../../components/DataGrid';
+import FEGGatewayContext from '../../components/context/FEGGatewayContext';
+import LoadingFiller from '@fbcnms/ui/components/LoadingFiller';
+import MagmaV1API from '@fbcnms/magma-api/client/WebClient';
+import React from 'react';
+import nullthrows from '@fbcnms/util/nullthrows';
+import useMagmaAPI from '@fbcnms/ui/magma/useMagmaAPI';
+
+import {
+  DynamicServices,
+  GatewayTypeEnum,
+  HEALTHY_STATUS,
+} from '../../components/GatewayUtils';
+import {
+  REFRESH_INTERVAL,
+  RefreshTypeEnum,
+  useRefreshingContext,
+} from '../../components/context/RefreshContext';
+import {useRouter} from '@fbcnms/ui/hooks';
+
+export default function GatewayDetailStatus({refresh}: {refresh: boolean}) {
+  const {match} = useRouter();
+  const networkId: string = nullthrows(match.params.networkId);
+  const gatewayId: string = nullthrows(match.params.gatewayId);
+  // Auto refresh gateways every 30 seconds
+  const refreshCtx = useRefreshingContext({
+    context: FEGGatewayContext,
+    networkId: networkId,
+    type: RefreshTypeEnum.FEG_GATEWAY,
+    interval: REFRESH_INTERVAL,
+    id: gatewayId,
+    refresh: refresh,
+  });
+  const fegGateways = refreshCtx.fegGateways || {};
+  const health = refreshCtx.health || {};
+  const gwInfo = fegGateways[gatewayId] || {};
+  let checkInTime;
+
+  if (
+    gwInfo.status &&
+    gwInfo.status.checkin_time !== undefined &&
+    gwInfo.status.checkin_time !== null
+  ) {
+    checkInTime = new Date(gwInfo.status.checkin_time);
+  }
+
+  const startTime = Math.floor(Date.now() / 1000);
+  const {response: cpuPercent, isLoading: isCpuPercentLoading} = useMagmaAPI(
+    MagmaV1API.getNetworksByNetworkIdPrometheusQueryRange,
+    {
+      networkId: networkId,
+      query: `cpu_percent{gatewayID="${gwInfo.id}", service="magmad"}`,
+      start: startTime.toString(),
+    },
+  );
+
+  const logAggregation =
```

Comment:
Rename this so it implies boolean value. Maybe something like `logAggregationEnabled`.